kumascript/macros/HTMLRefTable.ejs

```diff
@@ -28,31 +28,31 @@ let l10n = {
         "zh-CN": "描述",
         "ru"   : "Описание"
     }),
-    
+
```

Comment:
Sorry. I must have opened this file and pressed `⌘-S` which strips the whitespace. Oh well. What's done is done. It's not a terrible thing.